source/appModules/shellexperiencehost.py

```diff
@@ -16,3 +31,14 @@ def chooseNVDAObjectOverlayClasses(self,obj,clsList):
 				clsList.remove(ContentGenericClient)
 			except ValueError:
 				pass
+
+	# Argh, somehow, item status property repeats when Action Center is opened more than once.

```

Comment:
May be remove the word argh here to make this comment look somewhat more professional 😜